scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
That's right. And the fact that electron-builder doesn't support prefixed npm installs makes things worse. The npm version that we're using adds `optional: true` flags to shrinkwrap entries, so I think we can use that instead. I'll give it a go and see how it goes.